src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -2011,5 +2011,58 @@ private static bool TryFindFirstMatchedLane(Vector128<byte> mask, Vector128<byte
             matchedLane = BitOperations.TrailingZeroCount(selectedLanes) >> 2;
             return true;
         }
+
+        public static void ReverseByteRef(ref byte buf, nuint length)
```

Comment:
Nit: do these names need to include "ByteRef", "CharRef", etc., or can they all just be overloads of "Reverse", e.g. `public static void Reverse(ref byte buf, nuint length)`?

Reply:
Sure, I'll just overload `Reverse`.